jupyter-covid19/Dockerfile

```diff
@@ -7,10 +7,10 @@ RUN mkdir /home/jovyan/covid19-notebook
 
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/jhu-summary-overview/COVID-19-JHU_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
 ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/kaggle-demographics/kaggle_data_analysis_04072020.ipynb /home/jovyan/covid19-notebook/
-ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/feat/add_dependency/covid19-notebooks/readme.txt /home/jovyan/covid19-notebook/
-ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/feat/add_dependency/covid19-notebooks/requirements.txt /home/jovyan/covid19-notebook/
-ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/feat/add_dependency/covid19-notebooks/chicago-seir-forecast/covid19_seir.ipynb /home/jovyan/covid19-notebook/
-ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/feat/add_dependency/covid19-notebooks/chicago-seir-forecast/Img/seir_diagram.png /home/jovyan/covid19-notebook/
+ADD https://raw.githubusercontent.com/uc-cdis/covid19-tools/master/covid19-notebooks/readme.txt /home/jovyan/covid19-notebook/
```

Comment:
can we just copy the whole `covid19-tools/master/covid19-notebooks/` folder instead of one by one?